scripts/asm-installer/asm_vm

```diff
@@ -23,9 +23,9 @@ _CI_ISTIOCTL_REL_PATH="${_CI_ISTIOCTL_REL_PATH:=}"
 ### Internal variables ###
 MAJOR="${MAJOR:=1}"; readonly MAJOR;
 MINOR="${MINOR:=9}"; readonly MINOR;
-POINT="${POINT:=3}"; readonly POINT;
+POINT="${POINT:=5}"; readonly POINT;
 REV="${REV:=2}"; readonly REV;
-CONFIG_VER="${CONFIG_VER:="1-staging"}"; readonly CONFIG_VER;
+CONFIG_VER="${CONFIG_VER:="1-unstable"}"; readonly CONFIG_VER;
```

Comment:
This should be a config version right?

Reply:
Yes, this PR is just to kick the tests off. The actual release commit will be done via CLI to take care of these.

Reply:
Oh I see...